urth-viz-query.html

```diff
@@ -519,7 +493,8 @@
 
         ready: function() {
             // move filters template to user-supplied urth-core-dataframe in light DOM
-            var dataframe = this.$.myDataframe.getDistributedNodes()[0];
+            // var dataframe = this.$.myDataframe.getDistributedNodes()[0];
```

Comment:
Didn't mean to check this in here, but I'll clean it up and we can make this fix #87  too